### ISSUE TITLE
MultilineOperationIndentation bugs

### ISSUE THREAD

**Issue by bquorning** [rubocop/rubocop#1362]
I noticed a few bugs in the new MultilineOperationIndentation cop introduced in #1360:

Using the `indented` style, this is allowed:

``` rb
Foo
.a
  .b(c)
```

while this is reported as an offense:

``` rb
Foo
.a
  .b
```

Using the `aligned` style, this is allowed

``` rb
bar = Foo
  .a
      .b(c)

puts bar
```

while this is reported as an offense:

``` rb
bar = Foo
  .a
      .b

puts bar
```
